core/templates/dev/head/pages/exploration_editor/simple_editor_tab/simple_editor_sidebar_directive.html

```diff
@@ -112,6 +112,9 @@
   simple-editor-sidebar .highlighted {
     background-color: #c5e1a5;
   }
+  simple-editor-sidebar .error {
```

Comment:
Yes , i did and found out where and how material icons have been used. 
The thing is if i implement the icon feature in this issue , i need to work on finding out when the error status come up so that a scope variable $scope.isError is False and when error occurs it is set to true and using ng-show we can make to icon visible for the question with error .
So this will belong to a different issue as we need to write function which checks for the error .